src/ui/notification.js

```diff
@@ -6,8 +6,8 @@ const Notification = props => (
   <div
     sx={{
       ':not(:last-child)': { marginBottom: '1.5rem' },
-      backgroundColor: props.isDanger ? '#ff3860' : 'whitesmoke',
-      color: props.isDanger ? '#fff' : 'currentColor',
+      backgroundColor: props.isdanger ? '#ff3860' : 'whitesmoke',
+      color: props.isdanger ? '#fff' : 'currentColor',
       borderRadius: 2,
       padding: 3,
       position: 'relative'
```

Comment:
Regarding the renaming of `isDanger` to `isdanger`: the problem is that when you add all the props as attributes of this `div` below (`{...props}`), the `isDanger` property (which is not a valid HTML attribute) is added as well. By renaming it to `isdanger`, the warning is just silenced and we emit this invalid HTML attribute. The proper solution is to exclude `isDanger` from the properties that are added as HTML attributes. This works:

```js
const Notification = props => {
  const { isDanger, ...rest } = props;
  return (<div
    sx={{
      ':not(:last-child)': { marginBottom: '1.5rem' },
      backgroundColor: isDanger ? '#ff3860' : 'whitesmoke',
      color: isDanger ? '#fff' : 'currentColor',
      borderRadius: 2,
      padding: 3,
      position: 'relative'
    }}
    {...rest}
  />)
};
```

See [this StackOverflow Q&A](https://stackoverflow.com/questions/49358560/react-wrapper-react-does-not-recognize-the-staticcontext-prop-on-a-dom-elemen) for more information.

I think this should be changed (and of course, the renaming in the three other files should be reverted).

Reply:
Thank you for researching this.